lib/_debug_agent.js

```diff
@@ -45,18 +47,21 @@ exports.start = function start() {
 
 function Agent() {
   net.Server.call(this, this.onConnection);
+  this.maxConnections = 1;
```

Comment:
This prevent debugger from accepting more than one client. Please, read #858 to understand, why we need to limit clients number.